contrib/jinzhu/gorm/gorm.go

```diff
@@ -16,7 +16,7 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 
-	"github.com/jinzhu/gorm"
+	"gopkg.in/jinzhu/gorm.v1"
```

Comment:
Let’s not make any changes to this repo. We want to keep it as it is for older users.

Reply:
I have reverted the changes in this directory.